src/cljs/org/broadinstitute/firecloud_ui/common/input.cljs

```diff
@@ -24,8 +24,11 @@
                                :style (merge (or (:style props) {})
                                         (when (:invalid @state)
                                           {:borderColor (:exception-red style/colors)}))
-                               :onChange #(swap! state dissoc :invalid)
+                               :onChange #(do (swap! state dissoc :invalid)
+                                              (when-let [x (:onChange props)]
+                                                (x (.. % -target -value))))
```

Comment:
When an onChange handler has been defined in props (that is, the :onChange value in that map is not nil), call it with the value in the input field. onChange is passed the event, so (.. % -target -value) would be the same as JavaScript function(e) { return e.target.value; }

I maybe should have just passed the event along though to keep things consistent with normal input elements. That might be better.